README.md

```diff
@@ -24,8 +24,7 @@ The [master](https://github.com/butterproject/butter-desktop) branch which conta
 
 #### Quickstart:
 
-1. `npm install -g grunt-cli bower`
```

Comment:
please don't remove anything from the documentation. you're implementing new behaviour not removing any old one.